keylime/cloud_verifier_tornado.py

```diff
@@ -471,16 +471,16 @@ def start_tornado(tornado_server, port):
     print("Tornado finished")
 
 
-def main(argv=sys.argv):
+def start(port_key,db_key,rev_port_key):
     """Main method of the Cloud Verifier Server.  This method is encapsulated in a function for packaging to allow it to be
     called as a function by an external program."""
 
     config = configparser.ConfigParser()
     config.read(common.CONFIG_FILE)
 
-    cloudverifier_port = config.get('general', 'cloudverifier_port')
+    cloudverifier_port = config.get('general', port_key)
```

Comment:
There is no `port_key` on `keylime.conf` , is that a deliberate omission ?

Reply:
port_key is a var that's set by the caller to start.  See https://github.com/nabilschear/python-keylime/blob/be234cafb67e928cae20b685b9cd842e76565593/keylime/cloud_verifier_tornado.py#L521

Reply:
i've added three new config entries to keylime.conf: `provider_verifier_port`, `provider_db_filename`,`provider_revocation_notifier_port` . These get passed into the `start` routine by the new provider verifier main function: 
https://github.com/nabilschear/python-keylime/blob/be234cafb67e928cae20b685b9cd842e76565593/keylime/provider_verifier.py#L32

So, `port_key` and the others are just variables that point either to the old config entries or the new ones depending on who called start . This replicates the functionality of what was there when we only had one verifier.  
https://github.com/nabilschear/python-keylime/blob/be234cafb67e928cae20b685b9cd842e76565593/keylime/cloud_verifier_tornado.py#L521 

Maybe i should change the names of those variables to make it more clear?  (instead of xxx_key)